speed/inrix/process_cmp_in_virtual_env/cmpUpdateRunInConda.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+REM Set the path to the TOML file as the first argument
+SET TOML_FILE_PATH=%1
+
+REM Create the temporary conda environment
+CALL conda create --name temp_cmp_env -c conda-forge python=3.11 -y
```

Comment:
You mean the name of the virtual environment or the bat file? This virtual environment would be deleted after the processing.

Reply:
oh huh, maybe both.  And yep I got the deletion part but I just thought maybe it'd be clearer (though then the name I suggested is so long so it might just be worse).
Because CMP (update) usually means the report that comes out every 2 years, and just because of similarity we're using the same name for this monthly congestion tracker too.

Reply:
Got you, will do it!